### DISCUSSION TITLE
Compiling issue

### DISCUSSION THREAD

**Discussion by xtianpu** [gotd/td#1381] [Q&A]
Hello, I want to ask about the compiling my project with this lib, when I try this using go test it compiles my project until OOM manager triggered. This is my first time using a go lib that compiler need so many resources. Is there any way to solve this? Or do I missed some step?

My server specs are 2 vCPU and 2 GB RAM.

**Comment by AlienegraGeek** [comment]
It seems that your configuration is adequate, and when I ran it on my mac, it took up only a small portion of the resources
